python/ray/experimental/state/api.py

```diff
@@ -511,17 +511,30 @@ def summary(
         return summary_api_response["result"]["node_id_to_summary"]
 
 
-"""
-Convenient Methods for get_<RESOURCE> by id
-"""
-
-
 def get_actor(
     id: str,
     address: Optional[str] = None,
     timeout: int = DEFAULT_RPC_TIMEOUT,
     _explain: bool = False,
-) -> Optional[ActorState]:
+) -> Optional[Dict]:
+    """Get an actor by id.
+
+    Args:
+        id: Id of the actor
+        address: The URL of the API server. E.g., `http://localhost:8265`.
```

Comment:
Can you change all `address` saying it is the GCS address? (normally it is 6379 not 8265)